### ISSUE TITLE
ble-example: Can't establish connection

### ISSUE THREAD

**Issue by kelko** [lulf/microbit-bsp#16]
I tried the "ble" example from this repo. It started up, informed me that it began advertising, but I can't connect to it from my other devices (iMac, iPhone):
Neither of them sees the name of the device as defined in the example code ("Embassy Microbit"), but just display it as generic utility ("Zubehör" in the German UI). 
And when trying to pair a timeout occurs. Also no new debug info in the console while trying to pair.

(As per README I loaded the 7.3.0 version of the firmware onto the microbit.)

Do I need to perform any special steps to pair it with the smartphone, should that be possible at all or am I required to use some special tooling?

**Comment by jamessizeland** [comment]
It shouldn't require any special tooling on the phone side, nRF Connect app should work fine as a generic ble reader.

I'll try the example later and see.

**Comment by kelko** [comment]
OK, I did not know about the "nRF Connect App". It works with that one, I see the micro:bit and can connect to it.
Maybe it's worth adding that to the example doc?

**Comment by lulf** [comment]
Yeah, the reason it doesn't show up otherwise is probably some missing attributes. Using nrf connect is probably the best advise for this example anyway

**Comment by kelko** [comment]
> what are you trying to connect it to otherwise?

I just tried to connect to it from the iOS Bluetooth setting, and that did not work. (macOS bluetooth settings either)
And for now I just want to get to know how the whole thing works.
